net-libs/libwebsockets/libwebsockets-4.0.1-r1.ebuild

```diff
@@ -12,9 +12,8 @@ SRC_URI="https://github.com/warmcat/libwebsockets/archive/v${PV}.tar.gz -> ${P}.
 LICENSE="MIT"
 SLOT="0/16" # libwebsockets.so.16
 KEYWORDS="~amd64 ~arm ~x86"
-IUSE="access-log caps cgi client dbus generic-sessions http-proxy http2 ipv6
-	+lejp libev libevent libressl libuv peer-limits server-status smtp socks5
-	sqlite3 ssl static-libs threads zip"
+IUSE="access-log caps cgi client dbus +dir +file-ops generic-sessions http-proxy +http-uncommon-headers http2 ipv6 +lejp libev libevent libressl libuv peer-limits server-status smtp socks5 sqlite3 ssl static-libs +sys-capability testapps threads zip"
```

Comment:
+testapps.

Reply:
This like is annoying long.  I'm not actually sure what the policy is but consider breaking this up please.

I'm assuming sys-capability is this one:

caps - Use Linux capabilities library to control privilege.

I'm also inclined to say unless it pulls in additional dependencies (which I don't see that you've modified except for mbedtls).

There are others too.  Please always first look for use flags from ${PORTDIR}/profiles/use.desc for use.  Make sure they're the right ones.